### PR TITLE
aqbanking 6.2.1, gwenhywfar 5.3.0

### PR DESCRIPTION
**Pull request by SeekingMeaning**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Previous aqbanking PR: #56955

Previous gwenhywfar PR: #56954

/cc @fahasch

### CODE REVIEW

File: Formula/aqbanking.rb [hunk 1]
```diff
@@ -19,24 +19,23 @@ class Aqbanking < Formula
...
 
-  depends_on "pkg-config" => :build
   depends_on "gettext"
   depends_on "gmp"
   depends_on "gwenhywfar"
   depends_on "ktoblzcheck"
   depends_on "libxml2"
   depends_on "libxmlsec1"
   depends_on "libxslt"
+  depends_on "pkg-config"
```
Comment (chenrui333) [inline Formula/aqbanking.rb:29]: maybe `depends_on "pkg-config" => :build`?

Comment (chenrui333) [reply Formula/aqbanking.rb:29]: If it has to be runtime dependency, we might need to specify as comment.

Comment (SeekingMeaning) [reply Formula/aqbanking.rb]: Done ✅ Also reverted the minor whitespace change

**Review by chenrui333 — COMMENTED** [review]
lgtm. thanks!

**Review by SMillerDev — APPROVED** [review]
Thanks @SeekingMeaning ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Comment by SeekingMeaning** [comment]
(#56954 and #56955 can be closed)

**Pull request closed without merging**